src/SolveTree01.jl

```diff
@@ -96,6 +97,7 @@ function packFromLocalPartials!(fgl::G,
   #
 
   for idfct in getData(cliq).potentials
+    !(exists(fgl, idfct)) && (@warn "$idfct not in clique $(cliq.index)" continue)
```

Comment:
FIXME I just put this in without looking to get the code to run

Reply:
@dehann, I'm a bit concerned with this, it was a hack just to get code to run. It shouldn't be needed.